src/services/scheduler.ts

```diff
@@ -251,7 +251,7 @@ export async function start(dbId: string, dbUser: string) {
   }
 }
 
-async function getContext(conn: TypeormWrapper, Modules: any): Promise<Context> {
+export async function getContext(conn: TypeormWrapper, Modules: any): Promise<Context> {
```

Comment:
where do you need this?

Reply:
in the functional tests to build the context object.